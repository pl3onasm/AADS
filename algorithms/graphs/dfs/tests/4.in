14

0 4
0 11
0 5
1 4
1 8
1 2
2 5
2 6
2 9
3 2
3 6
3 13
4 7
5 8
5 12
6 5
8 7
9 10
9 11
10 13
12 9
