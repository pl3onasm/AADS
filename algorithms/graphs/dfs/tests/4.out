Discovery and finish times

   Node   Discovery    Finish
  label        time      time
      0           1        20
      1          21        26
      2          22        25
      3          27        28
      4          18        19
      5           2        17
      6          23        24
      7          14        15
      8          13        16
      9           4        11
     10           7        10
     11           5         6
     12           3        12
     13           8         9

Edge classification

  Starting   Ending      Edge
      node     node      type
         0        5         T
         0       11         F
         0        4         T
         1        2         T
         1        8         C
         1        4         C
         2        9         C
         2        6         T
         2        5         C
         3       13         C
         3        6         C
         3        2         C
         4        7         C
         5       12         T
         5        8         T
         6        5         C
         8        7         T
         9       11         T
         9       10         T
        10       13         T
        12        9         T
